checkout.py

```diff
@@ -3,11 +3,15 @@
 Created on Fri Aug 13 11:35:57 2021
 
 @author: vjham
+
 """
+from datetime import datetime
 from validation import is_clerk_input_valid
 
 items = []
 GROCERY_STORE_NAME = "Shop N Bag"
+DATE_AND_TIME = datetime.now()
+DATE_TIME_FORMAT = DATE_AND_TIME.strftime("%B %d %Y %H:%M:%S")
```

Comment:
Our date format IS a constant - so this seems fine to me, to keep all caps.

One thing I might do to make our lives easier in the long run is to add a comment at the end of a line with an example date (in case we forget how to read this format)

Something like # Example: Aug 29 2021 13:30:59
(or whatever the correct format is)